Makefile

```
@@ -127,8 +130,8 @@ clean:
 	@rm -rf $(FILES)
 
 .PHONY: goget-tools
-goget-tools:
-	(cd / && go install -mod=mod github.com/golangci/golangci-lint/cmd/golangci-lint@v$(GOLANGCI_LINT_VERSION))
+goget-tools:  ## Install binaries of the tools used by odo (current list: golangci-lint, mockgen)
+	(cd / && go install -mod=mod github.com/golangci/golangci-lint/cmd/golangci-lint@v$(GOLANGCI_LINT_VERSION) && go install -mod=mod github.com/golang/mock/mockgen@$(GO_MOCK_LIBRARY_VERSION))
```

nit: suggestion to improve readability (and also if we need to do additional things later):

```suggestion
	( \
		cd / && \
		go install -mod=mod github.com/golangci/golangci-lint/cmd/golangci-lint@v$(GOLANGCI_LINT_VERSION) && \
		go install -mod=mod github.com/golang/mock/mockgen@$(GO_MOCK_LIBRARY_VERSION) \
	)
```

---

or we can simply execute all those commands in separate shells:

```suggestion
	cd / && go install -mod=mod github.com/golangci/golangci-lint/cmd/golangci-lint@v$(GOLANGCI_LINT_VERSION)
	cd / && go install -mod=mod github.com/golang/mock/mockgen@$(GO_MOCK_LIBRARY_VERSION)
```